packages/frontend/src/features/scheduler/components/SchedulerPreview.tsx

```diff
@@ -14,7 +14,7 @@ import { CUSTOM_WIDTH_OPTIONS } from '../constants';
 
 type Props = {
     dashboard: Dashboard;
-    schedulerFilters: SchedulerFilterRule[] | undefined;
```

Comment:
Type suggestion related to another comment somewhere in my review that requested to keep this type for better distinguishing